rest-service/manager_rest/test/endpoints/test_plugins.py

```diff
@@ -272,8 +282,55 @@ def test_plugin_download_progress(self):
           client_max_version=base_test.LATEST_API_VERSION)
     def test_caravan_upload(self):
         self.upload_caravan(
-            {TEST_PACKAGE_NAME: TEST_PACKAGE_VERSION,
-             TEST_PACKAGE_NAME2: TEST_PACKAGE_VERSION2}
+            {
+                TEST_PACKAGE_NAME:
+                    [TEST_PACKAGE_VERSION, TEST_PACKAGE_YAML_FILE],
+                TEST_PACKAGE_NAME2:
+                    [TEST_PACKAGE_VERSION2, TEST_PACKAGE_YAML_FILE2]
+            }
         )
         plugins_list = self.client.plugins.list()
         self.assertEqual(len(plugins_list), 2)
+
+    @attr(client_min_version=3.1,
+          client_max_version=base_test.LATEST_API_VERSION)
+    def test_plugin_upload_with_empty_yaml(self):
```

Comment:
Change test name to indicate expected failure: test_plugin_upload_fail_with_empty_yaml